.eslintrc.js

```diff
@@ -1,10 +1,18 @@
 module.exports = {
     "env": {
         "browser": true,
-        "jquery": true
```

Comment:
Why did we remove jquery? This appears to perhaps be the cause of the issue:
https://travis-ci.com/internetarchive/bookreader/builds/117072482